build.gradle

```diff
@@ -39,8 +39,6 @@ sourceSets {
   }
     api {
         java {
-            srcDir 'src/cli/java/org/commcare/util/screen'
```

Comment:
Is this the result of a merge? I'm not sure I understand why removing these woudln't cause issues compiling the CLI jar

Reply:
Hm yes looks like a merge issue, sorry about that. I merged the latest and this resolved itself.